src/pages/informationmodel-details-page/information-model-details-page.jsx

```diff
@@ -329,11 +352,18 @@ const renderStickyMenu = (
 export const InformationModelDetailsPage = ({
   fetchPublishersIfNeeded,
   fetchReferenceDataIfNeeded,
+  fetchRdfDataIfNeeded,
   informationModelItem,
   publisherItems,
   referencedApis,
-  referenceData
+  referenceData,
+  rdfData
 }) => {
+  useEffect(() => {
+    // TODO: replace value with informationmodel id before merging
+    fetchRdfDataIfNeeded('1f44a866-2cbd-3b80-bf72-cccb99965e25');
+  }, []);
+
```

Comment:
- Either move all fetches into this useEffect or completely remove it
- Do not use hard-coded ID